src/js/providers/html5.js

```diff
@@ -363,7 +363,7 @@ define([
 
         function _pauseHandler() {
```

Comment:
What about
```js
 if (!_attached) {
    return;
}
```
?

I think there's a reason we're only looking at 'complete'. If the video enters the paused state it's active.